src/plots/xyPlot.ts

```diff
@@ -9,6 +9,19 @@ export class XYPlot<X, Y> extends Plot {
   private _adjustYDomainOnChangeFromXCallback: ScaleCallback<Scale<any, any>>;
   private _adjustXDomainOnChangeFromYCallback: ScaleCallback<Scale<any, any>>;
 
+  private _performanceEnabled = false;
+  private _fastPanZoomOnXCallback: ScaleCallback<Scale<any, any>>;
+  private _fastPanZoomOnYCallback: ScaleCallback<Scale<any, any>>;
+
+  private _fastPanZoomTimeoutReferenceX = 0;
+  private _fastPanZoomTimeoutReferenceY = 0;
+  private _fastPanZoomDeltaX = 0;
+  private _fastPanZoomDeltaY = 0;
+  private _fastPanZoomScaleX = 1;
+  private _fastPanZoomScaleY = 1;
+  private _fastPanZoomKnownDomainX: X[] = [null, null];
+  private _fastPanZoomKnownDomainY: Y[] = [null, null];
```

Comment:
A bunch of the above variables allude to some `_fastPanZoom` idea but nowhere in the below code references panning or zooming.  All it seems to be is scale domain updating.